doc/md/knowledge-center/mysql/generated_columns.md

```diff
@@ -0,0 +1,218 @@
+---
+id: mysql-generated-columns
+title: How to Use Generated Columns in MySQL with Atlas
```

Comment:
Generated columns are part of the SQL standard. i.e. support by ~all databases.

Can we make this doc more general and map the differences between dialects in one section? Another option is to use tabs in case grammar or examples are different between dialects (it's mostly the same). See examples in https://atlasgo.io/ddl/sql#schema.

Reply:
Hey , as we spoke on the phone this is on purpose to increase search visibility, we will have a specific article per DB

Reply:
OK, I understand from @rotemtam that it is going to be copied per dialect. My request please is to move it to a different page. Something like [entgo.io/tutorials](https://entgo.io/docs/tutorial-setup). 

<img width="660" alt="image" src="https://user-images.githubusercontent.com/7413593/177094954-1b185acd-47da-4a4d-ad1c-9c806a07caed.png">

Reply:
What is the decision on this?

Reply:
My opinion is that we should go ahead like this and create a dedicated section once we have more content. @a8m ?

Reply:
Let’s put it in a different category as it’s not part of the documentation and we prefer to start clean without tech debt. I plan to place the checks/lint rules in a separate category as well.

Please, let me know if you need help with this.

Reply:
I have moved the knowledge center to its own categorical page. My only concern is that it's called a Knowledge Center/Base but the URL is `/knowledge`. Rotem has moved the task of the page's stylization to a new pulll request.